common/src/main/java/net/opentsdb/query/QueryNodeConfig.java

```diff
@@ -0,0 +1,27 @@
+// This file is part of OpenTSDB.
+// Copyright (C) 2017  The OpenTSDB Authors.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 2.1 of the License, or (at your
+// option) any later version.  This program is distributed in the hope that it
+// will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser
+// General Public License for more details.  You should have received a copy
+// of the GNU Lesser General Public License along with this program.  If not,
+// see <http://www.gnu.org/licenses/>.
+package net.opentsdb.query;
+
+/**
+ * The configuration interface for a particular query node. Queries will populate
+ * the configs when instantiating a DAG.
+ * 
+ * @since 3.0
+ */
+public interface QueryNodeConfig {
```

Comment:
Will you be adding more methods to this interface?

Reply:
Possibly, for now it's just a placeholder with the ID.